tests/driver_at24cxxx/main.c

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (C) 2019 Otto-von-Guericke-Universität Magdeburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup tests
+ * @{
+ *
+ * @file
+ * @brief    AT24CXXX test application
+ *
+ * Tested with bluepill and AT24C256
+ *
+ * @}
+ */
+
+#include <errno.h>
+#include <stdio.h>
+#include <string.h>
+
+#include "xtimer.h"
+#include "board.h"
+
+#include "at24cxxx.h"
+
+#define I2C_DEVICE           I2C_DEV(0)
+
+#define WRITE_BYTE_POSITION  (12U)
+#define WRITE_BYTE_CHARACTER 'A'
+
+#define WRITE_POSITION       (444U)
+#define WRITE_CHARACTERS     { 'B', 'E', 'E', 'R', '4', 'F', 'R', 'E', 'E', \
+                               '\0' }
+
+#define SET_POSITION         (900U)
+#define SET_CHARACTER        'G'
+#define SET_LEN              (20U)
+
+int main(void)
+{
+    puts("Starting tests for module at24cxxx");
+    xtimer_init();
+    i2c_init(I2C_DEVICE);
+
+    at24cxxx_params_t params = {
+        .i2c = I2C_DEVICE,
+        .dev_addr = AT24CXXX_DEV_ADDR_00,
+        .pin_wp = GPIO_UNDEF,
+        .eeprom_size = AT24C256_EEPROM_SIZE
+    };
+
+    at24cxxx_t at24cxxx_dev;
+
+    int check;
+
+    /* Test: Init */
+    check = at24cxxx_init(&at24cxxx_dev, &params);
+    if (check != AT24CXXX_OK) {
+        printf("[FAILURE] init: (%d)\n", check);
+        return 1;
+    }
+    else {
+        puts("[SUCCESS] init");
+    }
+
+    /* Erase */
+    //at24cxxx_erase(&at24cxxx_dev);
```

Comment:
Forgot to remove the comment? I think the test should also test the return values of the functions.

Reply:
Actually I did not forget to remove the comment. Erasing takes a little time and as far as I know, EEPROM has a limit on write accesses before it breaks. Even if I had never gotten close to it, I didn´t want to push  it. I wanted to keep it as something someone could enable to exclude side effects from prior test runs.

I can of course remove the comment and add it to the test.

Reply:
OK, lets keep the the erase disabled. But maybe then just delete the two lines above to avoid confusing someone reading the code?